plugins/modules/redhat_subscription.py

```diff
@@ -40,6 +40,11 @@
         description:
             - access.redhat.com or Red Hat Satellite or Katello password
         type: str
+    token:
+        description:
+            - sso.redhat.com API access token.
+        type: str
+        version_added: 6.2.0
```

Comment:
Sorry, one last formality:
```suggestion
        version_added: 6.3.0
```